float64 cx
float64 cy
int32 id
float64 size
geometry_msgs/PoseStamped pose
